RestCommClient/Classes/RCDevice.m

```diff
@@ -119,6 +119,16 @@ - (id)initWithParams:(NSDictionary*)parameters delegate:(id<RCDeviceDelegate>)de
         self.reachabilityStatus = [_internetReachable currentReachabilityStatus];
         self.connectivityType = [RCDevice networkStatus2ConnectivityType:self.reachabilityStatus];
         
+        ICEConfigType iceConfigType = kXirsysV2;
+        //if ice config is not set, we will use
+        if ([parameters objectForKey:@"ice-config-type"]
+            && [[parameters objectForKey:@"ice-config-type"] intValue] >= 0
+            && [[parameters objectForKey:@"ice-config-type"] intValue] <= 2){
+            iceConfigType = (ICEConfigType)[[parameters objectForKey:@"ice-config-type"] intValue];
+        } else {
+            RCLogNotice("ice-config-type not found or invalid.");
```

Comment:
@ognjenns what happens if ice-config-type isn't provided? Are you somehow defaulting to V2?

Reply:
Yes, you can see it on line 122, by default its set to v2